src/app/features/quest/quest-preview/quest-preview.service.spec.ts

```diff
@@ -46,13 +46,15 @@ describe('QuestPreviewService', () => {
     const creatureQuestenderService = TestBed.inject(CreatureQuestenderService);
     const MAX_NEXT_QUEST_ID = 7;
 
-    spyOn(mysqlQueryService, 'getQuestTitleById').and.callFake(i => of('Title' + i).toPromise());
+    spyOn(mysqlQueryService, 'getQuestTitleById').and.callFake(i => Promise.resolve('Title' + i));
     spyOn(mysqlQueryService, 'getPrevQuestById').and.callFake(
-      (id: number) => of(String(id > 0 ? (id - 1) : 0)).toPromise()
+      (id: number) => Promise.resolve(String(id > 0 ? (id - 1) : 0))
     );
     spyOn(mysqlQueryService, 'getNextQuestById').and.callFake(
-      (id: number) => of(String(id < MAX_NEXT_QUEST_ID ? (id + 1) : 0)).toPromise()
+      (id: number) => Promise.resolve(String(id < MAX_NEXT_QUEST_ID ? (id + 1) : 0))
```

Comment:
these changes make the import of `of` no longer required

Reply:
removed